salt/modules/boto3_efs.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
If this works with boto2, we should create a new file for the boto3_efs module, instead of removing it.

This will cause people breakages that are using the boto2 version of this module.

Also, there is no need for the python hash bang at the top of this file.

Reply:
Removed hashbang